client/client-game-state-queue.cpp

```diff
@@ -0,0 +1,233 @@
+#include "client-game-state-queue.hpp"
```

Comment:
The indent on this file is 2 spaces, per the style guide it should be 4

Reply:
not sure I follow...

Reply:
if you are referring to how it's rendered on Github, that might just be a browser display issue, since all these files use TAB